<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContractVisitClass</class>
 <widget class="QWidget" name="ContractVisitClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1273</width>
    <height>889</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ContractVisit</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>1251</width>
     <height>771</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QTableView" name="tv_visits">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>271</y>
       <width>1251</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lb_tableTitle">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>229</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Current and future visits</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_addVisit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Visit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_deleteVisit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Visit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_quickSearch">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>421</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lb_quickSearch">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Visit Quick Search</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="gb_workerData">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>431</width>
       <height>281</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="title">
      <string>Worker Data</string>
     </property>
     <widget class="QLabel" name="lb_workerFirstName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="text">
       <string>Worker First Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_workerFirstName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_phone">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="lb_phone">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="text">
       <string>Phone</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_companyName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="text">
       <string>Company Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_companyName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_selectCompany">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>105</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="text">
       <string>Select Company</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_selectWorker">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Worker</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_addWorker">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_workerLastName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="text">
       <string>Worker Last Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_workerLastName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gb_additionalData">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>90</y>
       <width>641</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="title">
      <string>additional data</string>
     </property>
     <widget class="QLabel" name="lb_contactPerson">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="text">
       <string>Contact Person</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_contactPerson">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="toolTip">
       <string>If contact must be established differently.</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_activitiy">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="text">
       <string>Activity</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_activity">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="lb_location">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Location</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_location">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="te_note">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>361</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_note">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Note</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_reachableNote">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>150</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Reachable Note</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_reachableNote">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>170</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_status">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_status">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_contactPhone">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>230</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_contactPhone">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>210</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Contact Phone</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gb_data">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>400</y>
       <width>431</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Date</string>
     </property>
     <widget class="QDateTimeEdit" name="dte_arrivalAt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lb_arrivalAt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Arrival at</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_departureAt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Departure at</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dte_departureAt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_setArrivalAt">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_setDepartureAt">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>110</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableView" name="tv_visitWorker">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>400</y>
       <width>641</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_removeWorker">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>370</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Worker</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_saveVisit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>600</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Visit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_back">
     <property name="geometry">
      <rect>
       <x>1090</x>
       <y>20</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
